repo-agent/repowatch/controllers/repowatch_controller.go

```diff
@@ -1003,8 +1004,9 @@ func (r *RepoWatchReconciler) createSandboxForIssueHandler(ctx context.Context,
 					"name": handler.LLM.Provider,
 				},
 				"llm": map[string]interface{}{
-					"configdirRef": handler.LLM.ConfigdirRef,
-					"prompt":       prompt,
+					"configdirRef":     handler.LLM.ConfigdirRef,
```

Comment:
Similar to the comment above, consider renaming `apiKeySecretName` to `apiKeySecretRef` for consistency with the `RepoWatch` API.